Assets/ZSSRichTextEditor.js

```diff
@@ -710,6 +710,23 @@ ZSSEditor.getImageContainerNodeWithIdentifier = function(imageNodeIdentifier) {
     return $('#'+this.getImageContainerIdentifier(imageNodeIdentifier));
 };
 
+ZSSEditor.isMediaContainerNode = function(node) {
+    if (node.id === undefined) {
+        return false;
+    }
+    return (node.id.search("img_container_") == 0) || (node.id.search("video_container_") == 0);
+};
+
+ZSSEditor.extractMediaIdentifier = function(node) {
+    if (node.id.search("img_container_") == 0) {
+        return node.id.replace("img_container_", "");
+    }
+    
+    if (node.id.search("video_container_") == 0) {
```

Comment:
Since a single node should only have one identifier, why not place this condition in an else clause?  That way we will avoid evaluating both if "img_container_" is found.